bittensor/subtensor.py

```diff
@@ -4900,6 +4901,43 @@ async def neuron_for_uid_lite(
 
         return NeuronInfoLite.from_vec_u8(bytes_result)  # type: ignore
 
+    def decode_scale_bytes(self, return_type, scale_bytes, custom_rpc_type_registry):
```

Comment:
let's move `decode_scale_bytes` function to `bittensor/commands/utils.py` or `bittensor/commands/overview.py` as a helper function